addon/components/row-chart/style.less

```diff
@@ -1,23 +1,21 @@
 .row-chart {
-    .caret-icon {
-        font-family: FontAwesome;
-        font-size: 12px;
-    }
-
-    .chart-not-available {
-        text-shadow:
-            -1px -1px 0 #fff,  
-             1px -1px 0 #fff,
-             -1px 1px 0 #fff,
-             1px 1px 0 #fff;
-    }
-    path {
+    path,
+    .grid-line {
         fill: none;
-        stroke: @alto;
-        stroke-width: 1px;
+        stroke: @silver;
         shape-rendering: crispEdges;
     }
     .tick text {
         fill: @silver-chalice
     }
 }
+
+.d3-tip.e {
+    margin-left: 10px;
+}
+
+.d3-tip.e:after {
+    transform: rotate(90deg);
+    top: 50%;
+    left: -52%;
+}
```

Comment:
The way these styles are written right know they will apply to all charts. Is that intended?

Reply:
Yes, currently row chart is the only one that uses eastward tooltips. Tooltips are all actually rendered outside of charts, so you can't force it to apply only to one type of chart unless you add a class to the tooltip, but that shouldn't be necessary here. All eastward tooltips should be rotated like this regardless of the chart. Maybe then this code would be better in `tip.less`?